### PR TITLE
Fix Personal details output format

### PR DESCRIPTION
**Pull request by pabloai**
The personal details returns the output data in an outdated format, which is causing issues with Voucher components. This fixes this and aligns the functionality of the Company details component as well.

### CODE REVIEW

File: packages/playground/src/pages/OpenInvoices/OpenInvoices.js [hunk 1]
```diff
@@ -56,9 +56,9 @@ getPaymentMethods({ amount, shopperLocale }).then(paymentMethodsData => {
         .create('affirm', {
             countryCode: 'US', // 'US' / 'CA'
             visibility: {
-                personalDetails: 'editable', // editable [default] / readOnly / hidden
-                billingAddress: 'editable',
-                deliveryAddress: 'editable'
+                personalDetails: 'hidden', // editable [default] / readOnly / hidden
+                billingAddress: 'hidden',
+                deliveryAddress: 'hidden'
```
Comment (marcperez) [inline packages/playground/src/pages/OpenInvoices/OpenInvoices.js:61]: Hey @pabloai, are these changes intentional?

Comment (pabloai) [reply packages/playground/src/pages/OpenInvoices/OpenInvoices.js]: They weren't, thanks for checking! I just pushed the rollback.

File: packages/lib/src/components/Doku/Doku.tsx [hunk 2]
```diff
@@ -17,9 +17,9 @@ export class DokuElement extends UIElement {
      */
     formatData() {
         return {
+            ...this.state.data,
```
Comment (marcperez) [inline packages/lib/src/components/Doku/Doku.tsx:20]: Since we're refactoring this, could we change this to be more explicit on the fields that are returned?

File: packages/lib/src/components/Econtext/Econtext.tsx [hunk 3]
```diff
@@ -17,9 +17,9 @@ export class EcontextElement extends UIElement {
      */
     formatData() {
         return {
+            ...this.state.data,
```
Comment (marcperez) [inline packages/lib/src/components/Econtext/Econtext.tsx:20]: Same as with Doku, can we make this more explicit?

**Review by marcperez — APPROVED** [review]
